src/i18n/locales/it/translation.json

```diff
@@ -0,0 +1,610 @@
+{
+  "global": {
+    "loading": "Caricamento in corso",
+    "button_copy_text": "Copia",
+    "button_copy_text_confirmed": "Copiato",
+    "next": "Avanti",
+    "back": "Indietro",
+    "close": "Chiudi",
+    "abort": "Interrompi",
+    "table": {
+      "pagination": {
+        "items_per_page": {
+          "label": "Elementi per pagina",
+          "text_option_show_all": "Tutti"
+        },
+        "page_selector": {
+          "label_current": "Pagina",
+          "label_first": "Primo",
+          "label_previous": "Precedente",
+          "label_next": "Successivo",
+          "label_last": "Ultimo"
+        }
+      }
+    },
+    "errors": {
+      "error_loading_wallet_failed": "Caricamento del portafoglio fallito. {{ reason }}",
+      "error_reloading_wallet_failed": "Ricaricamento del portafoglio fallito. {{ reason }}",
+      "reason_unknown": "Motivo sconosciuto."
+    }
+  },
+  "app": {
+    "alert_no_connection": "Nessuna connessione al back-end: {{ connectionError }}"
+  },
+  "navbar": {
+    "title": "Jam",
+    "button_create_wallet": "Crea portafoglio",
+    "tab_send": "Invia",
+    "tab_receive": "Ricevi",
+    "tab_earn": "Guadagna",
+    "tab_sweep": "Sweep",
+    "joining_in_progress": "Disturbo",
+    "menu_mobile": "Menu",
+    "menu_mobile_settings": "Impostazioni"
+  },
+  "footer": {
+    "warning": "Questo software è in beta.<br /><1>Leggi qui prima di usarlo.</1>",
+    "warning_alert_title": "Avviso",
+    "warning_alert_text": "Mentre JoinMarket è collaudato e testato, Jam non lo è. È in fase beta, quindi usalo con cautela.",
+    "warning_alert_button_ok": "Accetto.",
+    "cheatsheet": "Riassunto",
```

Comment:
It makes sense, since the Cheatsheet it is composed by five basic and introductory steps, I would also consider "Istruzioni" (Instructions) and "Introduzione" (Introduction). What do you think?

Reply:
Let's go with "Introduzione" then :)